code/modules/mob/living/simple_animal/abnormality/waw/lindworm.dm

```diff
@@ -0,0 +1,210 @@
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "lindworm"
+	icon_living = "lindworm"
+	icon_dead = "lindworm_dead"
+	portrait = "lindworm"
+	del_on_death = FALSE
+	maxHealth = 700
+	health = 700
+	rapid_melee = 1
+	move_to_delay = 3
+	damage_coeff = list(RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1, PALE_DAMAGE = 1)
+	melee_damage_lower = 40
+	melee_damage_upper = 50
+	melee_damage_type = RED_DAMAGE
+	stat_attack = DEAD
+	attack_sound = 'sound/abnormalities/mountain/bite.ogg'
+	attack_verb_continuous = "gores"
+	attack_verb_simple = "gore"
+	can_breach = TRUE
+	threat_level = WAW_LEVEL
+	start_qliphoth = 4
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 60,
+		ABNORMALITY_WORK_INSIGHT = 60,
+		ABNORMALITY_WORK_ATTACHMENT = 60,
+		ABNORMALITY_WORK_REPRESSION = 60,
+	)
+	work_damage_amount = 10
+	work_damage_type = list(WHITE_DAMAGE, RED_DAMAGE)
+
+	ego_list = list(
+		/datum/ego_datum/weapon/vulnerability,
+		/datum/ego_datum/weapon/vulnerability,
+	)
+//	gift_type =  /datum/ego_gifts/harvest
+	abnormality_origin = ABNORMALITY_ORIGIN_ORIGINAL
+
+	color = "#234e99"
+	var/layers_left = 5
+	var/mob/living/carbon/human/red_weak
+	var/mob/living/carbon/human/white_weak
+	var/mob/living/carbon/human/black_weak
+	var/mob/living/carbon/human/pale_weak
+
+
+
+/mob/living/simple_animal/hostile/abnormality/lindworm/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH, PROC_REF(on_mob_death)) // Hell
+
+/mob/living/simple_animal/hostile/abnormality/lindworm/Destroy()
+	if(!layers_left)
+		UnregisterSignal(SSdcs, COMSIG_GLOB_MOB_DEATH)
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/lindworm/proc/on_mob_death(datum/source, mob/living/died, gibbed)
+	SIGNAL_HANDLER
+	if(!IsContained()) // If it's breaching right now
+		return FALSE
+	if(!ishuman(died))
+		return FALSE
+	if(died.z != z)
+		return FALSE
+	if(!died.mind)
+		return FALSE
+	datum_reference.qliphoth_change(-1) // One death reduces it
+	return TRUE
+
+/mob/living/simple_animal/hostile/abnormality/lindworm/death(gibbed)
+	for(var/turf/open/T in view(5, src))
+		if(locate(/obj/structure/turf_confetti) in T)
+			for(var/obj/structure/turf_confetti/floor_fire in T)
+				qdel(floor_fire)
+		new /obj/structure/turf_confetti(T)
+
+	playsound(get_turf(src), 'sound/abnormalities/bigbird/bite.ogg', 50, 1, 2)
+	new /obj/effect/gibspawner/generic/silent(get_turf(src))
+
+
+	if(layers_left)
+		layers_left-=1
+		HandleLayers()
+		revive(full_heal = TRUE, admin_revive = TRUE)
+		return
+
+	CleanupDefense()
+	density = FALSE
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/lindworm/AttackingTarget()
+	..()
+	for(var/turf/open/T in view(1, target))
+		if(locate(/obj/structure/turf_confetti) in T)
+			for(var/obj/structure/turf_confetti/floor_fire in T)
+				qdel(floor_fire)
+		new /obj/structure/turf_confetti(T)
+
+	if(ishuman(target))
```

Comment:
Your going to want to put a CanTarget override for non humans. Currently the stat attack applies to simplemobs and causes the lindworm to attack ordeal corpses.
Essentially just put in 
/mob/living/simple_animal/hostile/abnormality/lindworm/CanAttack(atom/the_target)
	if(isliving(the_target))
		var/mob/living/carbon/host = the_target
		if(host.stat == DEAD && !ishuman(the_target))
			return FALSE
	return ..()

Reply:
Actually it might be easier to do it backwards and remove the stat_attack and just put in a can_attack override for dead humans.

Reply:
Was fixed!